src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -11,13 +14,12 @@
  */
 public class Birthday implements DeepCopyable {
 
-    public static final String MESSAGE_CONSTRAINTS = "Birthdays should only contain numbers and forward slashes";
-
+    public static final String MESSAGE_CONSTRAINTS = "Birthdays should be in dd/mm/yyyy format with valid dates.";
```

Comment:
Can this instead say
`Birthdays should only contain numbers and forward slashes and must only take in valid months and days of respective months in dd/MM/yyyy` format